kfserving/kfserving-install/base/statefulset.yaml

```diff
@@ -39,7 +39,9 @@ spec:
               fieldPath: metadata.namespace
         - name: SECRET_NAME
           value: kfserving-webhook-server-secret
-        image: $(registry)/kfserving-controller:0.2.0
+        - name: ENABLE_WEBHOOK_NAMESPACE_SELECTOR
+          value: enabled
```

Comment:
Do we want this value to be configured by KfDef?

Reply:
yeah - enables kfserving deployment in kubeflow to be independent of standalone kfserving behaviour

Reply:
with kubeflow installation I think this should be always be on otherwise user is taking a risk. Once we migrate to object selector with kuberentes 1.15+ then this env can be retired.